### PR TITLE
Don't reset <ul>'s margin

### LINKED ISSUE #1550: [Wagtail: vertical spacing] Missing margin bottom in <ul> element
### Issue
Currently our `<ul>` element has no margin-bottom in the CMS. 

image

This element should be like our `<ol>` element (which has a margin-bottom of 1rem = 16px):

image

### PR DESCRIPTION
**Pull request by mmmavis**
Closes #1550

@youriwims just an example to show you what I meant in [my comment](https://github.com/mozilla/foundation.mozilla.org/pull/2452#issuecomment-454165463). Feel free to pick up this PR or start your own.

Let's just deal with margin for now and worry about other rules we have for `ul` in `resets.scss` next.

### CODE REVIEW

File: network-api/networkapi/buyersguide/templates/bg_base.html [hunk 1]
```diff
@@ -67,7 +67,7 @@
 
       <footer class="mofo-footer">
         <div class="container">
-          <ul class="list-unstyled footer-links row justify-content-center">
+          <ul class="list-unstyled footer-links row justify-content-center mb-0">
```
Comment (mmmavis) [inline network-api/networkapi/buyersguide/templates/bg_base.html]: `mb-0` (margin bottom 0): a bootstrap class.

Comment (youriwims) [reply network-api/networkapi/buyersguide/templates/bg_base.html]: Sounds good, I'll look through the project as well tomorrow.

**Comment by youriwims** [comment]
Based on #1550, @mmmavis & I removed the`margin: 0` `<ul>` reset rule so CMS `<ul>`are targeted with identical `margin-bottom` as `<ol>`. For non-CMS  `<ul>` (i.e. Footer & Fellowship page), we targeted those directly with `.mb-0` to remove that added `margin-bottom`.

**Comment by Pomax** [comment]
Good solution - fellowships are kind of limbo'd atm so making sure what they need is kept local makes a lot of sense.

**Comment by youriwims** [comment]
Fixed #2452

**Pull request merged**